pkg/acsengine/engine.go

```diff
@@ -723,20 +725,34 @@ func (t *TemplateGenerator) getTemplateFuncMap(cs *api.ContainerService) templat
 		"GetDCOSMasterCustomData": func() string {
 			masterProvisionScript := getDCOSMasterProvisionScript()
 			masterAttributeContents := getDCOSMasterCustomNodeLabels()
+			masterPreprovisionExtentsion := ""
```

Comment:
typo: `masterPreprovisionExtentsion` should be `masterPreprovisionExtension` (remove extra `t`)